src/notebook/notebookKernel.ts

```diff
@@ -167,11 +175,11 @@ export class JuliaKernel implements vscode.NotebookKernel {
 
             await serverListeningPromise.wait()
 
-            const jlexepath = await getJuliaExePath()
+            const jlexepath = this.executablePath
             const pkgenvpath = await getAbsEnvPath()
 
             this._terminal = vscode.window.createTerminal({
-                name: 'Julia Notebook Kernel',
+                name: `Julia Notebook Kernel ${path.basename(this.document.uri.fsPath)}`,
```

Comment:
This way if you have multiple notebooks you can tell which one is which.
Btw, what's the purpose of displaying this to the user, is it for diagnostic purposes (if things go wrong, they can see the output in the terminal)